mphys/core/multipoint.py

```diff
@@ -80,5 +105,30 @@ def mphys_add_scenario(self, name, scenario, coupling_nonlinear_solver=None,
         self.mphys_coupling_solvers.append((scenario, solver_tuple))
         return self.add_subsystem(name, scenario)
 
+    def mphys_connect_scenario_coordinate_source(self, source, scenarios, disciplines):
```

Comment:
Is this the same function definition as the one above? I'm not seeing any differences

Reply:
Should be removed  now